pom.xml

```diff
@@ -23,9 +23,30 @@
         <resilience4j.version>2.0.2</resilience4j.version>
         <reactor.kafka.version>1.3.16</reactor.kafka.version>
         <maven-checkstyle-plugin.version>3.2.2</maven-checkstyle-plugin.version>
+        <maven.surefire.plugin.version>3.1.0</maven.surefire.plugin.version>
+        <junit.version>5.9.3</junit.version>
+        <reactor.test.version>3.5.5</reactor.test.version>
     </properties>
 
     <dependencies>
+        <dependency>
```

Comment:
@eyakauleva test scoped dependencies should be located at the bottom of the `dependencies` XML node

Reply:
fixed 7fb4bb4